index.d.ts

```diff
@@ -1,3 +1,31 @@
+export interface TimeoutOptions {
+	/**
+	Milliseconds before timing out.
+
+	Passing `Infinity` will cause it to never time out.
+
+	@default Infinity
+	*/
+	milliseconds?: number;
+
+	/**
+	Specify a custom error message or error.
+
+	If you do a custom error, it's recommended to sub-class `pTimeout.TimeoutError`.
```

Comment:
It's not enough to just copy-paste. You also have to adapt to the current context.

```suggestion
	If you do a custom error, it's recommended to sub-class `TimeoutError`.
```